nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -71,16 +71,23 @@ def run_tvm_graph(graph_def, input_data, input_node, output_shape, output_dtype)
 def run_tf_graph(sess, input_data, input_node, output_node):
     """ Generic function to execute tensorflow """
 
-    tensor = sess.graph.get_tensor_by_name(output_node)
-
     if isinstance(input_data, list):
         input_dict = {}
         for i, e in enumerate(input_node):
             input_dict[e] = input_data[i]
     else:
         input_dict = {input_node: input_data}
 
-    output_data = sess.run(tensor, input_dict)
+    if isinstance(output_node, list):
+        output_list = []
+        for node_name in output_node:
```

Comment:
Can run one time by passing list of tensors to run method.

Reply:
Address the problem with update rework solution.